### DISCUSSION TITLE
Is there a way to port the items to another browser/profile?

### DISCUSSION THREAD

**Discussion by glenserraview** [gbraad/gauth#121] [Q&A]
I'm changing browsers and would like to port my existing configuration to the new browser. Is this possible?

Love the extension, thanks!

**Comment by seamusdemora** [comment]
What an odd forum...  a perfectly logical question open for 2 years without an answer! 

And I have the same question...  I'm a long-time user of this app, and thought I had exported my keys once before, but can now see no way to make that happen.

**Reply by bonifacio123** [reply]
It seems that functionality was approved and merged. Grab the repo and checkout out the master branch. It's there.

**Reply by seamusdemora** [reply]
> It seems that functionality was approved and merged. Grab the repo and checkout out the master branch. It's there.

Even if I did that, I wouldn't know how to restore `gauth` to "Extension status" in my browser... and I do like it in my browser. 

But, I now recall now how I did this... which I guess is/may be the only way to answer the OP's Question: 

1. open GAuth in a browser tab
2. select the pancake stack in the ULHC of the GAuth browser display
3. select "Settings"
4. click "Export keys", a new dialog will open asking where to save the `gauth-export.json` file; save it anywhere convenient
5. open the `gauth-export.json` file in your text editor (.json files are pure text files)
6. to "import", simply click the '+ Add' button in GAuth (in your new browser), and copy-paste the "acct name" & "Secret key" fields from the .json file, one-at-a-time.
